Sources/StreamChatUI/ChatThread/ChatThreadVC.swift

```diff
@@ -150,11 +155,16 @@ open class ChatThreadVC: _ViewController,
 
     // MARK: - ChatMessageListVCDataSource
 
+    // The replies data source.
     public var messages: [ChatMessage] {
         get {
             replies
```

Comment:
I have to admit that seeing returning replies for messages is a little bit confusing (i know this is not part of your changes just mentioning it)